src/scenes/AcceleratorRouter/Modules/Events.tsx

```diff
@@ -0,0 +1,222 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { Button, TableColumnType } from '@terraware/web-components';
+
+import Card from 'src/components/common/Card';
+import Table from 'src/components/common/table';
+import { APP_PATHS } from 'src/constants';
+import EventsCellRenderer from 'src/scenes/AcceleratorRouter/Modules/EventsCellRenderer';
+import strings from 'src/strings';
+import { Module, ModuleEvent } from 'src/types/Module';
+
+interface ModuleEventsProps {
+  module?: Module;
+  events?: ModuleEvent[];
+}
+
+const columns = (): TableColumnType[] => [
+  {
+    key: 'id',
+    name: strings.ID,
+    type: 'string',
+  },
+  {
+    key: 'status',
+    name: strings.STATUS,
+    type: 'string',
+  },
+  {
+    key: 'startTime',
+    name: strings.START_TIME,
+    type: 'string',
+  },
+  {
+    key: 'endTime',
+    name: strings.END_TIME,
+    type: 'string',
+  },
+  {
+    key: 'meetingUrl',
+    name: strings.MEETING_URL,
+    type: 'string',
+  },
+  {
+    key: 'recordingUrl',
+    name: strings.RECORDING_URL,
+    type: 'string',
+  },
+  {
+    key: 'slidesUrl',
+    name: strings.SLIDES_URL,
+    type: 'string',
+  },
+  {
+    key: 'projects',
+    name: strings.PROJECTS,
+    type: 'string',
+  },
+];
+
+export default function ModuleEvents({ events, module }: ModuleEventsProps): JSX.Element {
+  const theme = useTheme();
+  const navigate = useNavigate();
+
+  const liveSessions = events?.filter((ev) => ev.type === 'Live Session');
+  const oneOnOneSessions = events?.filter((ev) => ev.type === 'One-on-One Session');
+  const recordedSessions = events?.filter((ev) => ev.type === 'Recorded Session');
+  const workshops = events?.filter((ev) => ev.type === 'Workshop');
+
+  return (
+    <Card flushMobile style={{ display: 'flex', flexDirection: 'column', flexGrow: 1 }}>
+      <Box
```

Comment:
I forgot how to make this happen but we likely want the table to be scrollable instead of the whole card.